fiftyone/core/cli.py

```diff
@@ -2702,6 +2704,372 @@ def _print_operator_info(operator_uri):
     _print_dict_as_table(d)
 
 
+class DelegatedCommand(Command):
+    """Tools for working with FiftyOne delegated operations."""
+
+    @staticmethod
+    def setup(parser):
+        subparsers = parser.add_subparsers(title="available commands")
+        _register_command(subparsers, "launch", DelegatedLaunchCommand)
+        _register_command(subparsers, "list", DelegatedListCommand)
+        _register_command(subparsers, "info", DelegatedInfoCommand)
+        _register_command(subparsers, "cleanup", DelegatedCleanupCommand)
+
+    @staticmethod
+    def execute(parser, args):
+        parser.print_help()
+
+
+class DelegatedLaunchCommand(Command):
+    """Launches a service for running delegated operations.
+
+    Examples::
+
+        # Launch a local service
+        fiftyone delegated launch --type local
```

Comment:
@nebulae my thinking was to set this up so that something like this could be possible in the future:

```
fiftyone delegated launch --type airflow
```